pages/dkp/kommander/1.4/uninstall/index.md

```diff
@@ -21,6 +21,7 @@ For more details about uninstalling Konvoy, see the [Konvoy Documentation][konvo
 Managed clusters created by Kommander will not be deleted.
 Workloads, even if managed by Kommander, will continue to work as before but managing them centrally won’t be possible anymore.
 If you have an Identity Provider (IDP) configured, neither the login nor the kubeconfig that uses the IDP will work.
-You can still use the administrator kubeconfig, which can be loaded from the UI to access your managed cluster.
+You can still use the administrator kubeconfig, which can be [loaded from the UI to access your managed cluster][create-token-for-cluster].
 
+: /dkp/kommander/1.4/tutorials/login-cluster/
 [konvoy-uninstall]: /dkp/konvoy/1.8/uninstall
```

Comment:
The links still have some issues:

http://docs-d2iq-com-pr-3573.s3-website-us-west-2.amazonaws.com/dkp/kommander/1.4/uninstall/

Reply:
tried to fix this myself. Waiting for a rebuild to check it.

Reply:
Sorry - my bad. Thanks for checking!